src/SystemContainer_TEST.cc

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (C) 2024 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+#include <gtest/gtest.h>
+#include <unordered_set>
+#include "SystemCointainer.hpp"
```

Comment:
```suggestion
#include "SystemContainer.hpp"
```
after renaming the header file

Reply:
2416a19d571504576bbe8c7a5dfd40c63eaca13e